src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/AvailabilitySpecification.cs

```diff
@@ -28,8 +28,7 @@ public Decision IsSatisfiedBy(RemoteMovie subject, SearchCriteriaBase searchCrit
                     return Decision.Accept();
                 }
             }
-            //TODO: will need to handle the case for PreDB but for now, treat PreDB the same as Released
-            if (subject.Movie.Status < subject.Movie.MinimumAvailability || (subject.Movie.MinimumAvailability == MovieStatusType.PreDB && subject.Movie.Status < MovieStatusType.Released))
+            if (!subject.Movie.IsAvailable())
```

Comment:
@onedr0p : isAvailable method gets accessed here too! so it makes life really easy and the code much easier to maintain..